src/emulator/functionsEmulatorRuntime.ts

```diff
@@ -495,17 +572,18 @@ function ProtectEnvironmentalVariables(): void {
   process.env.GOOGLE_APPLICATION_CREDENTIALS = "";
 }
 
-function InitializeEnvironmentalVariables(projectId: string): void {
+function InitializeEnvironmentalVariables(frb: FunctionsRuntimeBundle): void {
+  const projectId = frb.projectId;
+
   process.env.GCLOUD_PROJECT = projectId;
   process.env.FUNCTIONS_EMULATOR = "true";
-  /*
-    Do our best to provide reasonable FIREBASE_CONFIG, based on firebase-functions implementation
-    https://github.com/firebase/firebase-functions/blob/master/src/index.ts#L70
-   */
+
+  // Do our best to provide reasonable FIREBASE_CONFIG, based on firebase-functions implementation
+  // https://github.com/firebase/firebase-functions/blob/master/src/index.ts#L70
```

Comment:
It's unclear to me what this comment (specifically the link into firebase-functions) means.

Reply:
Good call, updated the link to something better (and used a permalink this time)